gloss/sys_clock_gettime.c

```diff
@@ -0,0 +1,54 @@
+#define _POSIX_MONOTONIC_CLOCK 200809L
+#define _POSIX_TIMERS
+#include <errno.h>
+#include <metal/drivers/riscv_cpu.h>
+#include <metal/machine.h>
+#include <time.h>
+
+#ifdef MTIME_RATE_HZ_DEF
+#undef MTIME_RATE_HZ
+#define MTIME_RATE_HZ MTIME_RATE_HZ_DEF
+#endif
+
+#ifndef MTIME_RATE_HZ
+#define MTIME_RATE_HZ 32768
+#endif
+
+#ifdef __METAL_DT_RISCV_CLINT0_HANDLE
+#define mtime_interrupt_controller __METAL_DT_RISCV_CLINT0_HANDLE
+#endif
```

Comment:
We should also have a case here for the CLIC.

Reply:
I changed the code to look like this:
```
#if !defined(mtime_interrupt_controller) && defined(__METAL_DT_RISCV_CLINT0_HANDLE)
#define mtime_interrupt_controller __METAL_DT_RISCV_CLINT0_HANDLE
#endif

#if !defined(mtime_interrupt_controller) && defined(__METAL_DT_RISCV_CLIC0_HANDLE)
#define mtime_interrupt_controller __METAL_DT_RISCV_CLIC0_HANDLE
#endif
```